meta/run.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# In this file we specify how each microservice should boot.
+#
+
+runtime: native
```

Comment:
Are you intentionally using `native` here? I know we have an explicit `require` for Node.js because of the tutorial we have for using Capstan with no run configurations, but will it break if we specify `runtime: node` here?